src/Templates/NewModule/build.psake.ps1

```diff
@@ -375,6 +410,7 @@ function PromptUserForCredentialAndStorePassword {
 }
 
 function AddSetting {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSShouldProcess', '', Scope='Function')]
```

Comment:
BTW is this cropping up because we are analyzing the build script?  If so, perhaps we should only analyze scripts under the src dir?  Sorry if this is a duplicate comment.  I responded today by email (on my phone) while I was out and about with my wife.  Those email responses don't seem to get added to the review.  Oh well.